snuba/web/split.py

```diff
@@ -384,6 +387,11 @@ def execute(
         event_ids = list(
             set([event[self.__id_column] for event in result.result["data"]])
         )
+        if len(event_ids) > settings.COLUMN_SPLIT_MAX_RESULTS:
+            # We may be runing a query that is beyond clickhouse maximum query size,
+            # so we cowardly abandon.
+            return None
```

Comment:
Should this be tracked so we know how frequently it's occurring? This seems detrimental if we're doing it frequently.

Reply:
makes sense